float64[] prediction_input
---
int64[] prediction_output